raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -519,7 +535,7 @@ contract TokenNetwork is Utils {
     )
         view
         private
-        returns (uint256, uint256)
+        returns (uint256, uint256, uint256, uint256)
```

Comment:
not for this PR, but `getSettleTransferAmounts()` could use a docstring explaining at least what it returns